pkg/dashboard/assets/css/main.css

```diff
@@ -529,3 +529,79 @@ UTILITIES
   white-space: nowrap;
   width: 1px;
 }
+
+.email-box {
+  background: #4c415f;
+  padding: 1.375rem 0.8125rem;
+  width: 100%;
+}
+
+.email-box__toggle-title {
+  color: #fff;
+  font-size: var(--step-4);
+  font-weight: 600;
+}
+
+.email-box__content p {
+  color: #fff;
+  padding: 0.75rem 0 1.5rem 0;
+}
+
+.email-box__email-label {
+  position: relative;
+}
+
+.email-box__email-label span {
+  color: #d3d3d3;
+  font-weight: 400;
+  left: 0;
+  left: 0.5rem;
+  position: absolute;
+  top: -8%;
+}
+
+.email-box__email-input {
+  background: #fff;
+  border-radius: 0.25rem;
+  border: 1px solid #d3d3d3;
+  margin-bottom: 1rem;
+  padding: 0.5rem;
+  width: 71%;
+}
+
+.email-box__email-input:focus-visible {
+  outline: thick solid #4d90fe;
+}
+
+.email-box__checkbox-label {
+  color: #fff;
+  display: block;
+  font-size: 1rem;
+}
+
+#email-box__checkbox {
+  accent-color: #445688;
+}
+
+.email-box__privacy {
+  color: #4d90fe !important;
+}
+
+.email-box__submit-btn {
+  background-color: #c4c4c4;
+  border-radius: 0.25rem;
+  border: none;
+  color: #000;
+  font-weight: 600;
+  margin-top: 1rem;
+  padding: 8px 24px;
+}
+
+.email-box__submit-btn:hover {
+  cursor: pointer;
+}
+
+.email-box__submit-btn--active {
+  background-color: #BAECF5;
+  color: #fff;
```

Comment:
I noticed it's hard to see both the active button's text and its `outline` when focused. Could we please increase the contrast of both of these things so they're more visible?

Reply:
I updated the code. Thank you for your feedback!

Reply:
This change made a big difference. Thank you!